<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>411</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="TabWidgetPage1">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QLabel" name="InputLabel1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Input 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="InputLabel2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Input 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="InputLabel3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Input 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="InputLabel4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Input 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="InputLabel5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Input 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="InputLabel6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Input 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="InputLabel7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Input 1</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="StartDateSelect">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="EndDateSelect">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="EndDateLabel">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>End Date</string>
      </property>
     </widget>
     <widget class="QLabel" name="StartDateLabel">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Date</string>
      </property>
     </widget>
     <widget class="QLabel" name="Status">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
     <widget class="QComboBox" name="InputBox1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>311</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="InputBox2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>311</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="InputBox3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>311</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="InputBox4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>311</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="InputBox5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>311</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="InputBox16">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>290</y>
        <width>311</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="InputBox7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>330</y>
        <width>311</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="InputBox18">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TabWidgetPage2">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
